application/github/v0/config/tasks.json

```diff
@@ -0,0 +1,1607 @@
+          "items": {
+            "instillFormat": "string",
+            "instillAcceptFormats": [
+              "string"
+            ],
+            "instillUpstreamTypes": [
+              "value",
+              "reference"
+            ],
+            "type": "string"
+          }
+        },
+        "labels": {
+          "description": "Labels of the issue",
+          "instillUIOrder": 8,
+          "title": "Labels",
+          "type": "array",
+          "items": {
+            "instillFormat": "string",
+            "instillAcceptFormats": [
+              "string"
+            ],
+            "instillUpstreamTypes": [
+              "value",
+              "reference"
+            ],
+            "type": "string"
+          }
+        },
+        "is_pull_request": {
+          "description": "Whether the issue is a pull request",
+          "instillUIOrder": 9,
+          "title": "Is Pull Request",
+          "instillFormat": "boolean",
+          "instillAcceptFormats": [
+            "boolean"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "type": "boolean"
+        }
+      },
+      "title": "Issue",
+      "type": "object"
+    }
+  },
+  "TASK_GET_ALL_PULL_REQUESTS": {
+    "instillShortDescription": "Get the list of all pull requests in a repository",
+    "input": {
+      "description": "Please input the repository name and owner",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [
+        "owner",
+        "repository"
+      ],
+      "properties": {
+        "$ref": "#/$defs/repository_info",
+        "state": {
+          "default": "open",
+          "title": "State",
+          "description": "State of the PRs, including open, closed, all. Default is open",
+          "enum": [
+            "open",
+            "closed",
+            "all"
+          ],
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 10,
+          "type": "string"
+        },
+        "sort": {
+          "default": "created",
+          "title": "Sort",
+          "description": "Sort the PRs by created, updated, popularity, or long-running. Default is created",
+          "enum": [
+            "created",
+            "updated",
+            "popularity",
+            "long-running"
+          ],
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 11,
+          "type": "string"
+        },
+        "direction": {
+          "default": "desc",
+          "title": "Direction",
+          "description": "Direction of the sort, including asc or desc. Default is asc",
```

Comment:
NIT
Typo?
`"description": "Direction of the sort, including asc or desc. Default is asc",`
Default is desc